qcodes/dataset/sqlite/queries.py

```diff
@@ -99,6 +99,8 @@ def _build_data_query(table_name: str,
     return query
 
 
```

Comment:
Since we are putting deprecation warnings qcodes.dataset.data_set. The same warning in queries is not needed.

Reply:
Well I am not sure, I just deprecated when I see one. Having two warnings will not be a problem, if it tails, IMHO.